homeassistant/components/deconz/__init__.py

```diff
@@ -44,8 +44,8 @@
 SERVICE_DATA = 'data'
 
 SERVICE_SCHEMA = vol.Schema({
-    vol.Exclusive(SERVICE_FIELD, 'deconz_id'): cv.string,
-    vol.Exclusive(SERVICE_ENTITY, 'deconz_id'): cv.entity_id,
+    vol.Optional(SERVICE_FIELD, 'deconz_id'): cv.string,
```

Comment:
Change order to have entity listed before field, alphabetical order. This will also help with the description later on.

Reply:
Use matches_regex to make sure that field starts with an '/'

Reply:
All right. Will do.